docs/tutorial.rst

```diff
@@ -237,7 +237,110 @@ c) Select a few antenna pairs to keep
+****************
+
+Basic options are 'auto', 'cross', or 'all'.  The ant_str can also contain:
+
+1. Individual antenna number(s):
+________
+
+- 1: returns all antenna pairs containing antenna number 1 (including the auto correlation)
+- 1,2: returns all antenna pairs containing antennas 1 and 2
+
+::
+
+  from pyuvdata import UVData
+  UV = UVData()
+  filename = 'pyuvdata/data/day2_TDEM0003_10s_norx_1src_1spw.uvfits'
+  UV.read_uvfits(filename)
+  # Print all the antennas numbers with data in the original file
+  print(UV.get_antpairs())
+  # Apply select to UV object
+  UV.select(ant_str='1,2,3')
+  # Print all the antennas numbers with data after the select
+  print(UV.get_antpairs())
+
+2. Individual baseline(s):
+________
+
+- 1_2: returns only the antenna pair (1,2)
+- 1_2,1_3,1_10: returns antenna pairs (1,2),(1,3),(1,10)
+- (1,2)_3: returns antenna pairs (1,3),(2,3)
+- 1_(2,3): returns antenna pairs (1,2),(1,3)
+
+::
+
+  from pyuvdata import UVData
+  UV = UVData()
+  filename = 'pyuvdata/data/day2_TDEM0003_10s_norx_1src_1spw.uvfits'
+  UV.read_uvfits(filename)
+  # Print all the antennas numbers with data in the original file
+  print(UV.get_antpairs())
+  # Apply select to UV object
+  UV.select(ant_str='(1,2)_(3,4)')
+  # Print all the antennas numbers with data after the select
+  print(UV.get_antpairs())
+
+3. Antenna number(s) and polarization(s):
+________
+
+When polarization information is passed with antenna numbers,
+all antenna pairs kept in the object will retain data for each specified polarization
+
+- 1x: returns all antenna pairs containing antenna number 1 and polarizations xx and xy
+- 2x_3y: returns the antenna pair (2,3) and polarization xy
+- 1r_2l,1l_3l,1r_4r: returns antenna pairs (1,2), (1,3), (1,4) and polarizations rr, ll, and rl.  This yields a complete list of baselines with polarizations of 1r_2l, 1l_2l, 1r_2r, 1r_3l, 1l_3l, 1r_3r, 1r_11l, 1l_11l, and 1r_11r.
+- (1x,2y)_(3x,4y): returns antenna pairs (1,3),(1,4),(2,3),(2,4) and polarizations xx, yy, xy, and yx
+- 2l_3: returns antenna pair (2,3) and polarizations ll and lr
+- 2r_3: returns antenna pair (2,3) and polarizations rr and rl
+- 1l_3,2x_3: returns antenna pairs (1,3), (2,3) and polarizations ll, lr, xx, and xy
+- 1_3l,2_3x: returns antenna pairs (1,3), (2,3) and polarizations ll, rl, xx, and yx
+
+::
+
+  from pyuvdata import UVData
+  UV = UVData()
+  filename = 'pyuvdata/data/day2_TDEM0003_10s_norx_1src_1spw.uvfits'
+  UV.read_uvfits(filename)
+  # Print all the antennas numbers and polarizations with data in the original file
+  print(UV.get_antpairs(), UV.get_pols())
+  # Apply select to UV object
+  UV.select(ant_str='1r_2l,1l_3l,1r_4r')
+  # Print all the antennas numbers and polarizations with data after the select
+  print(UV.get_antpairs(), UV.get_pols())
+
+4. Stokes parameter(s):
+________
+
+Can be passed lowercase or uppercase
+
+- i,I: keeps only Stokes I
+- q,V: keeps both Stokes Q and V
```

Comment:
Is there a file distributed with pyuvdata that contains Stokes params?  The file I used for the other examples only has circular pols.